app/views/markdown/help.md

```diff
@@ -1 +1,81 @@
+
+
+## Frequently Asked Questions 
+- [Who can use ScholarSphere?](#who-can-use-scholarsphere)
+- [Does it cost anything to upload to ScholarSphere?](#does-it-cost-anything-to-upload-to-scholarsphere)
+- [What can I deposit to ScholarSphere?](#what-can-i-deposit-to-scholarsphere)
+- [Can I deposit my scholarly article in ScholarSphere?](#can-i-deposit-my-scholarly-article-in-scholarsphere)
+- [Can I get a DOI (Digital Object Identifier) for my deposited work?](#can-i-get-a-doi-digital-object-identifier-for-my-deposited-work)
+- [How can I update or remove my work?](#how-can-i-update-or-remove-my-work)
+- [What happens to my files in ScholarSphere after I leave Penn State?](#what-happens-to-my-files-in-scholarsphere-after-i-leave-penn-state)
+- [Are materials deposited to ScholarSphere kept forever?](#are-materials-deposited-to-scholarsphere-kept-forever)
+- [Are there limits to how much I can deposit to ScholarSphere?](#are-there-limits-to-how-much-i-can-deposit-to-scholarsphere)
+- [How can I deposit very large datasets that are too large to upload through the web browser?](#how-can-i-deposit-very-large-datasets-that-are-too-large-to-upload-through-the-web-browser)
+- [What's the difference between ScholarSphere and cloud services like Dropbox?](#whats-the-difference-between-scholarsphere-and-cloud-services-like-dropbox)
+
+#### Who can use ScholarSphere? 
+
+*   The content in ScholarSphere is open to the public and free for anyone to search and download.
+*   To deposit your work in ScholarSphere, you must be a Penn State faculty, student, or staff, with a current Web Access Account ID and password. Penn State emeriti faculty may also use ScholarSphere.
+
+#### Does it cost anything to upload to ScholarSphere? 
+
+*   No, the University provides this service at no cost to current Penn State faculty, students, and staff and to emeriti faculty.
+
+#### What can I deposit to ScholarSphere? 
+
+*   All types of scholarly materials, including publications, instructional materials, creative works, and research data can be deposited to ScholarSphere To be eligible for deposit in ScholarSphere, works must:
+1. Be produced or sponsored by at least one member of the Penn State community;
+2. Be of scholarly import, including publications, instructional materials, creative works, and research data produced in support of Penn State's teaching, learning, and research mission; and
+3. Meet the requirements of[ Penn State Policy AD69 Accessibility of Electronic and Information Technology](https://policy.psu.edu/policies/ad69).
+*   Information with a Sensitive Information Classification of Moderate (Level 2) or higher according to[ Penn State Policy AD95 Information Assurance and IT Security](https://policy.psu.edu/policies/ad95) is not permitted in ScholarSphere.
+*   For more, see[ ScholarSphere’s Content and Deposit Policy](https://scholarsphere.psu.edu/about).
```

Comment:
One thing I might do, if this is possible in markdown, is any internal links to Scholarsphere routes, I might make those as relative paths rather than full URLs.

i.e. `/about` instead of `https://scholarsphere.psu.edu/about`

That way these links will still work when deployed to other hosts, such as beta/staging/preview/etc. 

As I mentioned, I do not know if Markdown actually supports this, I think it would be worth trying. If that's not possible, feel free to ignore this